lib/components/Search.dart

```diff
@@ -27,35 +36,20 @@ class Search extends StatelessWidget {
         hintText: "Try to find...",
         hintStyle:
             $styles.text.subheading1.copyWith(color: $styles.colors.grey),
-        prefixIcon: SizedBox(
-          width: 20,
-          height: 20,
+        prefixIcon: SizedBox.square(
+          dimension: 20,
           child: Center(
             child: FaIcon(
               FontAwesomeIcons.magnifyingGlass,
               color: $styles.colors.black,
             ),
           ),
         ),
-        focusedBorder: OutlineInputBorder(
-            borderSide: BorderSide(
-              color: $styles.colors.black,
-              width: $styles.borderStyle.width,
-            ),
-            borderRadius: $styles.borderStyle.circularBorderRadius),
-        enabledBorder: OutlineInputBorder(
-            borderSide: BorderSide(
-              color: $styles.colors.black,
-              width: $styles.borderStyle.width,
-            ),
-            borderRadius: $styles.borderStyle.circularBorderRadius),
+        focusedBorder: border,
+        enabledBorder: border,
         //  Weird that just setting this didn't work
-        border: OutlineInputBorder(
-            borderSide: BorderSide(
-              color: $styles.colors.black,
-              width: $styles.borderStyle.width,
-            ),
-            borderRadius: $styles.borderStyle.circularBorderRadius),
+        // What did not work? We should file a bug.
```

Comment:
What is not working here? Can we file a bug to investigate?

Reply:
@leighajarett I don't believe I touched the Search code! You probably have better insight than me!

Reply:
I think I was confused that "border" didn't set the border for all the different states but seems like it's intended - I can still file a bug to track it

Reply:
Added here: https://github.com/flutter/flutter/issues/140331